azure/postgresql/terraform/modules/postgres/variables.tf

```diff
@@ -0,0 +1,112 @@
+### PostgreSQL server variables
+
+#Resource group name for potgresql server
+variable "postgres_resource_group_name" {
+  type = string
+  description = "Name of resource group to hold postgres"
+}
+#Resource group tags for potgresql server
+variable "postgres_resource_group_tags" {
+  type = map
+  description = "Tags to be added to the postgres resource group"
+}
+#Resource group location for potgresql server
+variable "postgres_resource_group_location" {
+  type = string
+  description = "The location of the resource group which will hold postgres"
+}
+#Name for postgres server
+variable "postgres_name" {
+  type = string
+  description = "Name for postgres server"
+}
+#Location of postgres server
+variable "postgres_location" {
+  type = string
+  description = "Location of the resource group"
+}
+#Version for postgres server
+variable "postgres_version" {
+  type = string
+  description = "Version of porstgres"
+}
+variable "postgres_tags" {
+  type = map
+}
+#Database storage in megabytes
+variable "storagemb" {
+  type = string
+  description = "Amount of storage for the database, in mb."
+  default = "5120"
+}
+#Number of days to retain backups
+variable "backup_retention_days" {
+  type = number
+  default = 7
+}
+#Enable geo redundency for Azure postgres
+#Enable geo redundancy for Azure postgres
+variable "geo_redundent_enabled" {
+  type = bool
+  description = "Geo redundent postgres enablement, currently supports only: true"
+  default = true
+}
+#Enable auto grow for Azure postgres
+variable "auto_grow_enabled" {
+  type = bool
+  default = true
+  description = "Auto grow postgres enablement, currently supports only: true"
+}
+#Postgres sku name (GP_Gen5_2)
+variable "sku_name" {
+  type =string
+  description = "Sku name for postgres, currently supports only: General and Memory optimizes tiers"
+  default = "GP_Gen5_2"
+}
+#Name for database in postgres
+variable "database_name" {
+  type = string
+  description = "Name for database in postgres"
+}
+#Login for the database
+variable "database_login" {
+  type = string
+  description = "Login for server and database admin"
+}
+#Password for the database
+variable "database_password" {
+  type = string
+  description = "Password for the database in postgres"
```

Comment:
Perhaps use the `sensitive` value for the credentials variables?
https://learn.hashicorp.com/tutorials/terraform/sensitive-variables#refactor-database-credentials